docs/headless-cms/docs/06-02-clusterdocstopic.md

```diff
@@ -0,0 +1,84 @@
+---
+title: ClusterDocsTopic
+type: Custom Resource
+---
+
+The `clusterdocstopics.cms.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an orchestrator that creates ClusterAsset CRs for a specific asset type. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd clusterdocstopics.cms.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample ClusterDocsTopic custom resource (CR) that provides details of the ClusterAsset CRs for the **redis**, **asyncapi**, **markdown**, and **openapi** source types.
+
+```
+piVersion: cms.kyma-project.io/v1alpha1
```

Comment:
`a` got eaten :)
anyway I don't like this example as it is the same as the example from DocsTopic, I'd like to have here the example that makes sense, that introduced example with `view-context: docs-ui`, the example that I provided and we use already in our documentation CRs (you can check with @magicmatatjahu